src/CovidTracker/plot_time_series.py

```diff
@@ -14,4 +34,56 @@ def plot_ts(df):
     Examples
     ----------
     >>> plot_ts(covid_df)
```

Comment:
example function call missing metric input 
ie should say something like plot_ts(covid_df, 'cases')

Reply:
fixed, thanks!